File: app/src/main/java/bassamalim/hidaya/database/daos/TelawatDao.java
package bassamalim.hidaya.database.daos;

import androidx.room.Dao;
import androidx.room.Query;

import java.util.List;

import bassamalim.hidaya.database.dbs.TelawatDB;

@Dao
public interface TelawatDao {

    @Query("SELECT * FROM telawat_reciters NATURAL JOIN telawat_versions")
    List<TelawatDB> getAll();

}

File: app/src/main/java/bassamalim/hidaya/database/daos/TelawatDao.kt
package bassamalim.hidaya.database.daos

import androidx.room.Dao
import androidx.room.Query
import bassamalim.hidaya.database.dbs.TelawatDB

@Dao
interface TelawatDao {
    @get:Query("SELECT * FROM telawat_reciters NATURAL JOIN telawat_versions")
    val all: List<TelawatDB>
}